packages/salesforcedx-utils-vscode/test/jest/telemetry/telemetry.test.ts

```diff
@@ -91,4 +91,47 @@ describe('Telemetry', () => {
       expect(result).toBe(telemetryService.extensionName);
     });
   });
+
+  describe('Telemetry Service - getUserId', () => {
```

Comment:
Nit: to following the established naming convention this should just be the method under test.